themes/V3/5ePHB/snippets.js

```diff
@@ -212,6 +212,21 @@ module.exports = [
 						}}
 						\n`;
 				},
+			},
+			{
+				name : 'Homebrewery Credit',
+				icon : 'fas fa-dice-d20',
+				gen  : function(){
+					return dedent`
+						{{homebreweryCredits
+						Made With
+						
+						{{homebreweryIcon,gold}}
+						
+						Homebrewery.Naturalcrit.com
```

Comment:
Two things:  1) Should this be a link to the homebrewery?  2)  NaturalCrit should have the C capitalized as well.

Reply:
Dealing with 2) first, I have just pushed a commit that capitalizes the C.
On 1), this is an interesting question. Should this be a link (and thus https://homebrewery.naturalcrit.com) or just the name of the project ("The Homebrewery"), or both ([The Homebrewery](https://homebrewery.naturalcrit.com))??? As previously mentioned, this is currently a duplicate of the bottom of the back cover, where it exists as the text of the link, but not actually a link.

Reply:
Actually, having the URL (homebrewery.NaturalCrit.com) is helpful for print documents.  But it should also be a link.

Reply:
I've updated it now, to this:

![image](https://github.com/naturalcrit/homebrewery/assets/19826659/cb308972-b2e1-4479-b873-5ca17e8d4046)

The website address is a link, it responds to mouse over with an underline:

![image](https://github.com/naturalcrit/homebrewery/assets/19826659/43316941-0c51-4875-994a-98de6ae8e6b8)